velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -325,11 +331,15 @@ TEST_F(RegexFunctionsTest, regexpReplacePosition) {
 }
 
 TEST_F(RegexFunctionsTest, regexpReplaceNegativePosition) {
```

Comment:
nit: Can you add a test where you have bad unicode chars in the string and ensure behavior is similar to spark?